### PR TITLE
feat: add YourControls discord server command

### PR DESCRIPTION
**Pull request by Awemeter**
## Description

This PR adds a command for the YourControls discord server
## Test Results
image

## Discord Username
Awemeter#3110

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -141,3 +141,4 @@ Update <small>_ February 2022</small>
 - fix: whois offline status (28/02/2022)
 - fix: node_modules error (23/02/2022)
 - feat: update to discord.js v13 and API v9 (23/02/2022)
+- feat: add YourControls discord server command (15/08/2022)
```
Comment (benw202) [inline .github/CHANGELOG.md:144]: This is not in the correct place. Please move it to the appropriate month

File: .github/command-docs.md [hunk 2]
```diff
@@ -95,7 +95,7 @@
 | .when             | Explain the absence of release dates or ETAs                                                                            | ---                                 |
 | .thumb            | Answers the big question, will it have FEATURE?                                                                         | .willithave                         |
 | .xbox             | Short response + link to NOTAM for xbox marketplace                                                                     | .xboxmarketplace                    |
-
+| .yourcontrols     | Provides link to the YourControls discord                                                                               | .yc                                 |
```
Comment (benw202) [inline .github/command-docs.md:98]: You need a new line between this and Utilities

File: src/commands/general/yourcontrols.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourcontrols: CommandDefinition = {
+    name: ['yourcontrols', 'yc'],
+    description: 'Provides link to the YourControls Discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
```
Comment (benw202) [inline src/commands/general/yourcontrols.ts:9]:
```suggestion
        return msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
```

File: src/commands/general/yourcontrols.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourcontrols: CommandDefinition = {
```
Comment (benw202) [inline src/commands/general/yourcontrols.ts:4]:
```suggestion
export const yourControls: CommandDefinition = {
```

File: src/commands/index.ts [hunk 5]
```diff
@@ -134,6 +134,7 @@ import { docsearch } from './general/docsearch';
 import { dlss } from './support/dlss';
 import { temporarycommandedit } from './moderation/temporaryCommandEdit';
 import { temporarycommand } from './general/temporaryCommand';
+import { yourcontrols } from './general/yourcontrols';
```
Comment (benw202) [inline src/commands/index.ts:137]:
```suggestion
import { yourControls } from './general/yourcontrols';
```

File: src/commands/index.ts [hunk 6]
```diff
@@ -270,6 +271,7 @@ const commands: CommandDefinition[] = [
     dlss,
     temporarycommandedit,
     temporarycommand,
+    yourcontrols,
```
Comment (benw202) [inline src/commands/index.ts:274]:
```suggestion
    yourControls,
```

We will be refactoring to use camelCase at some point, so starting now eases our pain later haha!

File: src/commands/general/yourcontrols.ts (hunk 3 above)
Comment (benw202) [reply src/commands/general/yourcontrols.ts:9]: Is this a link you know won't get deleted? (as in talked to admins so they don't prune it)

**Review by benw202 — DISMISSED** [review]

File: src/commands/general/yourcontrols.ts (hunk 3 above)
Comment (Awemeter) [reply src/commands/general/yourcontrols.ts:9]: > Is this a link you know won't get deleted? (as in talked to admins so they don't prune it)

This is the link they have on their github

File: src/commands/general/yourcontrols.ts [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourControls: CommandDefinition = {
+    name: ['yourcontrols', 'yc'],
+    description: 'Provides link to the YourControls Discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        return msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
```
Comment (benw202) [inline src/commands/general/yourcontrols.ts:9]:
```suggestion
        await msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
        return msg.delete();
```

File: src/commands/general/yourcontrols.ts [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourControls: CommandDefinition = {
+    name: ['yourcontrols', 'yc'],
+    description: 'Provides link to the YourControls Discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
+        return msg.delete();
```
Comment (NathanInnes) [inline src/commands/general/yourcontrols.ts:10]: Why delete the user message?

File: src/commands/general/yourcontrols.ts [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourControls: CommandDefinition = {
+    name: ['yourcontrols', 'yc'],
+    description: 'Provides link to the YourControls Discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
```
Comment (NathanInnes) [inline src/commands/general/yourcontrols.ts:9]: Do they have a vanity link?

File: src/commands/general/yourcontrols.ts (hunk 8 above)
Comment (Awemeter) [reply src/commands/general/yourcontrols.ts:10]: idk ben told me to

File: src/commands/general/yourcontrols.ts (hunk 9 above)
Comment (Awemeter) [reply src/commands/general/yourcontrols.ts:9]: Nope

Comment (NathanInnes) [reply src/commands/general/yourcontrols.ts:9]: Okay will mark as resolved. 👍🏼

File: src/commands/general/yourcontrols.ts (hunk 8 above)
Comment (NathanInnes) [reply src/commands/general/yourcontrols.ts:10]: @benw202

Comment (benw202) [reply src/commands/general/yourcontrols.ts:10]: Was done on another invite link, MSFS, feel free to remove on both :)

Comment (NathanInnes) [reply src/commands/general/yourcontrols.ts:10]: No need to delete user message, please remove async function also as it is unnecessary. :)

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**